src/routes/PaymentCheckPage.tsx

```diff
@@ -155,7 +155,11 @@ export default function PaymentCheckPage() {
       setPayLoading(false);
       window.removeEventListener("beforeunload", onBrowserUnload);
       const url = new URL(authorizationUrl);
-      navigate(`${url.pathname}${url.hash}`);
+      if (url.hostname.includes("checkout.pagopa.it")) {
```

Comment:
Mhh. Maybe it's better to test if a url is "esternal" using `origin` ?
Something like
` if url.origin == window.location.origin`

Reply:
you're right, fixed with https://github.com/pagopa/pagopa-checkout-fe/pull/325/commits/fae093b6209440447fb435a9d69e3bd784be74eb